fvtest/compilertest/tests/LogFileTest.cpp

```diff
@@ -0,0 +1,262 @@
+/*******************************************************************************
+ *
+ * (c) Copyright IBM Corp. 2016, 2016
+ *
+ *  This program and the accompanying materials are made available
+ *  under the terms of the Eclipse Public License v1.0 and
+ *  Apache License v2.0 which accompanies this distribution.
+ *
+ *      The Eclipse Public License is available at
+ *      http://www.eclipse.org/legal/epl-v10.html
+ *
+ *      The Apache License v2.0 is available at
+ *      http://www.opensource.org/licenses/apache2.0.php
+ *
+ * Contributors:
+ *    Multiple authors (IBM Corp.) - initial implementation and documentation
+ *******************************************************************************/
+
+#include "tests/LogFileTest.hpp"
+
+#include <stdio.h>
+#include <string.h>
+#include <fstream>
+#include <string>
+#include <sstream>
+#include <unistd.h>
+#include <sys/stat.h>
+
+#include "tests/OMRTestEnv.hpp"
+#include "tests/OpCodesTest.hpp"
+
+TestCompiler::LogFileTest::LogFileTest()
+   {
+   // Don't use fork(), since that doesn't let us initialize the JIT
```

Comment:
In general, please replace uses of `JIT` with `compiler` throughout this file and the commit message.